pizzakit-plugin/includes/pizzakit.php

```diff
@@ -28,39 +29,44 @@ private static function handle_post() {
 		// In this the example it's "pizzakitFormSubmission".
 		if (isset($data["pizzakitFormSubmission"])) {
 
-			// insert the data into the different tables
-			// This assumes that the data is in a map-structure with the column names as keys
-			insert_into_tables($data);
+			Pizzakit::insert_into_tables($data);
 
 			// Respond with a JSON object by calling
 			// wp_send_json($response);
 			// where $response is an associative array.
+		}
 	}
 
-
 	private static function insert_into_tables($_data){
 
-		//insert into orders
-		$sql = "INSERT INTO wp_orders(email, name, telNr, address, doorCode, postalCode)
-			VALUES ("
-				. $_data['email'] . ", "
-				. $_data['name'] . ", "
-				. $_data['telNr'] . ", "
-				. $_data['address'] . ", "
-				. $_data['doorCode'] . ", "
-				. $_data['postalCode'] . ",)";
+		global $wpdb;
+
+		// Get the ID to use for this order
+		$_result = $wpdb->get_results("SELECT MAX(id)+1 AS maxID FROM wp_orders", OBJECT);
+		$_orderID	= $_result[0]->maxID;
 
+		//insert into orders
+		$sql = "INSERT INTO wp_orders(id, email, name, telNr, address, doorCode, postalCode, comments) VALUES ("
+				. $_orderID . ", '"
+				. $_data["email"] . "', '"
+				. $_data["name"] . "', '"
+				. $_data["telNr"] . "', '"
+				. $_data["address"] . "', '"
+				. $_data["doorCode"] . "', '"
+				. $_data["postalCode"] . "', '"
+				. $_data["comments"] . "')";
```

Comment:
[Prepare](https://developer.wordpress.org/reference/classes/wpdb/prepare/) the SQL statements to avoid SQL injections.